scripts/development/build-contenta_jsonapi.sh

```diff
@@ -52,9 +51,10 @@ if [ $? -ne 0 ]; then
 fi
 
 cd ${DEST_DIR}
+
 $COMPOSER config repositories.contenta_jsonapi path ${BASE_DIR}
 
-$COMPOSER require "contentacms/contenta_jsonapi:*" "phpunit/phpunit:~4.8" --no-progress
+$COMPOSER require "contentacms/contenta_jsonapi:*" "phpunit/phpunit:~5.7" --no-progress
```

Comment:
The reasoning  behind the PHPUnit Version update is that Version 4.8 is no longer supported since Feb 3 2017.